torch/nn/modules/adaptive.py

```diff
@@ -150,12 +155,17 @@ def __init__(
             osz = self.cutoffs[i + 1] - self.cutoffs[i]
 
             projection = Sequential(
-                Linear(self.in_features, hsz, bias=False),
-                Linear(hsz, osz, bias=False)
+                Linear(self.in_features, hsz, bias=False,
+                       reset_parameters=reset_parameters, **factory_kwargs),
+                Linear(hsz, osz, bias=False,
+                       reset_parameters=reset_parameters, **factory_kwargs),
             )
 
             self.tail.append(projection)
 
+        if reset_parameters:
+            self.reset_parameters()
```

Comment:
This seems wrong; you don't have to explicitly reset parameters here because the propagated reset parameters arguments should have handled it already

Reply:
Yep, good catch! fixed